style/js/screen_gfx.js

```diff
@@ -20,5 +20,5 @@ export function getGfxContext(bounds) {
  * So this function will always return 1 or 2 as appropriate based on DPR.
  */
 export function getPixelRatio() {
-  return window.devicePixelRatio > 1 ? 2 : 1;
+  return (typeof window !== "undefined" && window.devicePixelRatio) > 1 ? 2 : 1;
```

Comment:
This change seems unrelated?

Reply:
Without this change, the script throws an error:

```
file:///path/to/openstreetmap-americana/style/js/screen_gfx.js:23
  return window.devicePixelRatio > 1 ? 2 : 1;
  ^

ReferenceError: window is not defined
    at Module.getPixelRatio (file:///path/to/openstreetmap-americana/style/js/screen_gfx.js:23:3)
    at file:///path/to/openstreetmap-americana/style/js/shield_canvas_draw.js:10:24
    at ModuleJob.run (node:internal/modules/esm/module_job:197:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:337:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:61:12)
```

It’s because of this function call being exported as part of a module that gets imported:

https://github.com/ZeLonewolf/openstreetmap-americana/blob/21916204118f3da81e2cafe51185b53bd17d33f6/style/js/shield_canvas_draw.js#L10